Sources/DronecodeSDKSwiftDemo/Controllers/MapViewController.swift

```diff
@@ -10,6 +10,7 @@ class MapViewController: UIViewController, CLLocationManagerDelegate {
     @IBOutlet weak var mapView: MKMapView!
     @IBOutlet weak var feedbackLabel: UILabel!
     @IBOutlet weak var uploadMissionButton: UIButton!
+	@IBOutlet var cancelMissionUploadButton: UIButton!
```

Comment:
Indenting?

Reply:
This was missed and I see I didn't conform to what was previously done by using weak.